lib/bundler/dsl.rb

```diff
@@ -218,6 +219,14 @@ def method_missing(name, *args)
 
   private
 
+    def gem_helpers
```

Comment:
What about throwing a `module_function` at the top of `GemHelpers` and calling it directly?